src/environments/environment.ts

```diff
@@ -28,4 +30,6 @@ export const environment = {
   appVersion: "0.0.0", // replaced automatically during docker build
   repositoryId: "Aam-Digital/ndb-core",
   remoteLoggingDsn: undefined, // only set for production mode in environment.prod.ts
+  demo_mode: true,
```

Comment:
environment.ts and AppConfig with its static properties is kinda serving the same purpose, which might make it confusing for people where to put a new option. We could consider overwriting environment.ts properties during bootstrap and throughout the app always import and use environment.ts only. But in the end this might be too much hidden magic ... just a crazy thought I wanted to put out here.